out/artifacts/trivialTrivia_Web_exploded/js/App.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * module controls logic for Trivia api form submission,
+ * @returns {boolean}
+ * @constructor
+ */
+export const TriviaApi = () => {
+    event.preventDefault();
+
+    // get search form from dom
+    const searchForm = document.getElementById("form-api").elements;
+
+    // create url with params from form elements
+    const url = "/TrivialTrivia/api/trivia/request/?" + $.param({
+        triviaAmount: searchForm.namedItem("triviaAmount").value,
+        triviaCategory: searchForm.namedItem("triviaCategory").value,
+        triviaDifficulty: searchForm.namedItem("triviaDifficulty").value,
+        triviaType: searchForm.namedItem("triviaType").value,
+    });
+
+    // // send request for trivia, pass results to results showResults
+    // fetch(url, {headers: {'Content-type': 'application/json; charset=UTF-8', 'Accept': 'application/json'}})
+    //     .then(response => response.json()) // parse response as json
+    //     .then(data => showResults(data))// pass data to call back function
+    //     .catch(error => alert("Error sending request, try again." + " url: " + url));
```

Comment:
Are we going to need this later or can we delete the commented out code?

Reply:
yes this will be needed for the fetch request to the api, for now i commented it out and used test data to finish building the rest of the script, once the api is ready I will switch it out and we can merge this branch in